CppCoreGuidelines.md

```diff
@@ -15168,7 +15168,7 @@ Testing - often to an extreme extent - is essential.
 
 ##### Enforcement
 
-Have strong rules for re-testing in place that covers any change in hardware, operating system, compiler, and libraries.
```

Comment:
We don't think "strong" and "weak" are exclusive language (e.g., "strong attractor"), so would you please remove this and the next five changes, thanks!

Reply:
reverted